src/app/components/Document/index.jsx

```diff
@@ -27,10 +26,6 @@ const Document = ({ assets, app, data, styleTags, helmet }) => {
   return (
     <html lang="en-GB" {...htmlAttrs}>
       <head>
-        <meta httpEquiv="X-UA-Compatible" content="IE=edge" />
-        <meta charSet="utf-8" />
-        <meta name="robots" content="noindex,nofollow" />
-        <meta name="theme-color" content={C_POSTBOX} />
         {meta}
         <link rel="shortcut icon" href="/favicon.ico" type="image/x-icon" />
```

Comment:
Could the favicon one move over too? Seem pretty universal

Reply:
Yes it could be shared. However it isn't Metadata in that it isn't a `<meta>` tag, so I didn't want to add this to the Metadata component.  Separately to this PR, we'll want to handle the favicon and serviceworker links.

Reply:
I'm not sure I understand this. "Universal"? Most of the things we've moved seem to be the component seem to be the non-universal things... 🤔